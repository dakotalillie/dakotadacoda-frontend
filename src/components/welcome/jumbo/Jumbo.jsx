import React from 'react';
import { Container, Jumbotron } from 'reactstrap';
import githubIcon from '../../../img/social_icons/white/github.png';
import mediumIcon from '../../../img/social_icons/white/medium.png';
import linkedinIcon from '../../../img/social_icons/white/linkedin.png';
import socialLinks from '../../../utils/links.json';
import './jumbo.css';

class Jumbo extends React.Component {
  render() {
    return (
      <div className="jumbo">
        <Jumbotron fluid>
          <Container>
            <h1 className="display-3">Dakota Lillie</h1>
            <p className="lead">Software Developer</p>

            <div className="social_icons">
              <a href={socialLinks.github}>
                <img src={githubIcon} alt="Github" />
              </a>
              <a href={socialLinks.medium}>
                <img src={mediumIcon} alt="Medium" />
              </a>
              <a href={socialLinks.linkedin}>
                <img src={linkedinIcon} className="linkedin" alt="Linked In" />
              </a>
            </div>
          </Container>
        </Jumbotron>
      </div>
    );
  }
}

export default Jumbo;
