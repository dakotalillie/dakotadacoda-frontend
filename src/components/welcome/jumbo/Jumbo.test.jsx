import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Jumbo from './Jumbo';

describe('Jumbo', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Jumbo />);
  });

  it('renders successfully', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});