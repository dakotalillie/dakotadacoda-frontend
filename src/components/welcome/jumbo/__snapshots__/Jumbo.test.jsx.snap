// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jumbo renders successfully 1`] = `
<div
  className="jumbo"
>
  <Jumbotron
    fluid={true}
    style={
      Object {
        "paddingTop": 198,
      }
    }
    tag="div"
  >
    <Container
      tag="div"
    >
      <h1
        className="display-3"
      >
        Dakota Lillie
      </h1>
      <p
        className="lead"
      >
        Software Developer
      </p>
      <div
        className="social_icons"
      >
        <a
          href="https://github.com/dakotalillie"
        >
          <img
            alt="Github"
            src="github.png"
          />
        </a>
        <a
          href="https://medium.com/@dakota.lillie"
        >
          <img
            alt="Medium"
            src="medium.png"
          />
        </a>
        <a
          href="https://www.linkedin.com/in/dakotalillie/"
        >
          <img
            alt="Linked In"
            className="linkedin"
            src="linkedin.png"
          />
        </a>
      </div>
    </Container>
  </Jumbotron>
</div>
`;
